README.rst

```diff
@@ -0,0 +1,29 @@
+###############
```

Comment:
I hadn't noticed you'd moved this to the root. Please don't do this: we have a perfectly good readme for the repository already.

Reply:
Reverted.